rfc-113-expanding-draft-access-for-unauthenticated-users.md

```diff
@@ -0,0 +1,212 @@
+  only accessible by particular users, users in a particular organisation or
+  by users that have a token to access them.
+- **step-by-step:** A particular document type on GOV.UK which is
+  both a page in it's own right and a collection of pages that comprise
+  the guidance ([example](https://www.gov.uk/learn-to-drive-a-car)).
+
+[govuk-draft]: https://draft-origin.publishing.service.gov.uk
+[link expansion]: https://github.com/alphagov/publishing-api/blob/0ed88b14193f85b1b85b495a7c2617493b3c3ed2/doc/link-expansion.md
+[signon]: https://github.com/alphagov/signon
+[Content Store]: https://github.com/alphagov/content-store
+[Asset Manager]: https://github.com/alphagov/asset-manager
+
+## Problem
+
+GOV.UK editors produce step-by-steps that need fact checking by people who
+may not have a GOV.UK signon account. As step-by-steps can span multiple
+pages on GOV.UK the process to fact check them involves not only
+reviewing the step-by-step navigation page but also reviewing the pages that
+are part of the step-by-step.
+
+The current process to provide unauthenticated access to a GOV.UK draft is to
+use an auth-bypass token which allows access to a single content item. This is
+not sufficient to allow the preview of all the pages that are part of a
+step-by-step.
+
+Preparing a step-by-step for fact check is currently done by someone manually
+saving each draft page to a HTML files and then wiring them up into
+a small Heroku application. This application is then shared with the
+fact checker. This is a cumbersome and time consuming process, that we believe
+can be improved.
+
+## Proposal
+
+In considering how this problem can be resolved questions were raised about
+ownership of resources (content and assets) and the rights editors have to
+grant, or restrict, access on the draft stack of GOV.UK. Three publication
+states were considered - live, draft and access limited -  and their privacy
+expectations were defined:
+
+- **Live**: This resource is publicly available on live GOV.UK, there are no
+  privacy expectations.
+- **Draft**: This resource is available to users with GOV.UK signon accounts,
+  access is restricted from the wider internet however it is not considered
+  private.
+- **Access limited**: this resource is available to users as the result
+  of implicit permissions granted at the editors discretion (for example,
+  limiting to an organisation in Whitehall) and a limited number of GOV.UK
+  admins. There is an expectation that this resource will be kept private
+  from users who do not meet the criteria.
+
+These were then used to form conclusions on the level of access a
+step-by-step editor should be able to grant to a fact checker.
+
+The editor should be able to grant access to a fact checker to view the
+draft step-by-step navigation page via an auth-bypass token, this should allow
+the fact checker to bypass access limits on the content and any assets created
+for it.
```

Comment:
Should this be "this **must not** allow the fact checker to bypass access limits on the content"?  Later you say

> If there is a need to preview access limited content as part of a fact check
> then individual links will have to be provided for that content.

Or is "access limits" here being used to mean "things which are draft", and not "things which are access limited"?

Reply:
As the step-by-step navigation page is the document that the editor owns (in constrast to the linked pages which the editor does not own) an auth-bypass token provides access to that page whether or not there are any access limits for the content. What the editor cannot do is provide a token that can bypass access limits on any content they do not own.

Reply:
Ok, that makes sense.